expressions/aggregation/AggregationHandleAvg.hpp

```diff
@@ -109,8 +129,41 @@ class AggregationHandleAvg : public AggregationConcreteHandle {
     ++state->count_;
   }
 
+  inline void iterateUnaryInlFast(const TypedValue &value,
+                                  std::uint8_t *byte_ptr) const {
+    DCHECK(value.isPlausibleInstanceOf(argument_type_.getSignature()));
+    if (value.isNull()) return;
+    TypedValue *sum_ptr =
+        reinterpret_cast<TypedValue *>(byte_ptr + blank_state_.sum_offset_);
+    std::int64_t *count_ptr =
+        reinterpret_cast<std::int64_t *>(byte_ptr + blank_state_.count_offset_);
+    *sum_ptr = fast_add_operator_->applyToTypedValues(*sum_ptr, value);
+    ++(*count_ptr);
+  }
+
+  inline void updateState(const std::vector<TypedValue> &arguments,
```

Comment:
We only need one argument here, instead of a vector. As a further optimization, we can use an Untyped value (i.e. a pointer) instead of a TypedValue. As the fast_add_operator_ in this class is already an UncheckedOperator, it can work with untyped values.